projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -123,6 +125,18 @@ export default function PublicizeForm( {
 		[ dismissNotice, NOTICES ]
 	);
 
+	const pageHasFocus = usePageHasFocus();
+	const refreshOptions = useDispatch( SOCIAL_STORE_ID ).refreshAutoConversionSettings;
+
+	const isAutoConversionEnabled = useSelect(
+		select => select( SOCIAL_STORE_ID ).isAutoConversionEnabled(),
+		[]
+	);
+
+	if ( pageHasFocus ) {
+		refreshOptions();
```

Comment:
I think you had an idea about only checking the option if we need to. Could we do this only if we're going to render the notice with the 'change setting' link? Then if someone turns it off we'll only update once, and if someone dismisses the notice, we won't update at all.

Reply:
Yeah good point, I missed adding that. Added.!